components/modules/crypto.c

```diff
@@ -32,21 +36,21 @@ static const algo_info_t algorithms[] = {
     { "SHA256",    32, MBEDTLS_MD_SHA256 },
     { "SHA384",    64, MBEDTLS_MD_SHA384 },
     { "SHA512",    64, MBEDTLS_MD_SHA512 },
+#ifdef CONFIG_NODEMCU_CMODULE_SODIUM
+    { "BLAKE2b",     0, SODIUM_BLAKE2B }, // 0 because no fixed size for BLAKE2b hashes
+#endif
 };
 
 
+const int MAX_HASH_SIZE = 64; // Must be >= all the sizes in the above array
```

Comment:
No fixed size for blake2b, but 64 max? I'd say stick the 64 into the `algorithms` array entry then. Less risk of this one getting overlooked if a larger hash gets added at some later time.